packages/fiori/src/Wizard.js

```diff
@@ -731,6 +753,18 @@ class Wizard extends UI5Element {
 		return this.ariaLabel || this.i18nBundle.getText(WIZARD_NAV_ARIA_ROLE_DESCRIPTION);
 	}
 
+	get effectiveStepSwitchThreshold() {
+		if (this.stepSwitchThreshold < 0.5) {
```

Comment:
You can use the new `base/util/clamp.js` function for this in the future, but don't change it now